tests/ci/stress_check.py

```diff
@@ -183,6 +188,13 @@ def run_stress_test(docker_image_name):
     state, description, test_results, additional_logs = process_results(
         result_path, server_log_path, run_log_path
     )
+
+    if timeout_expired:
```

Comment:
We rather should have a simple function in `report.py` for it

Reply:
Like it's implemented for [BuildResult](https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/report.py#L418-L420). So, `TestResult` could have static method `check_timeout`

Reply:
Or  `run_timeout`